README.md

```diff
@@ -54,6 +80,7 @@ Here is a list of backends known to me.
 
 List of backends for geolocation:
 * [AppleWifiNlpBackend](https://github.com/microg/AppleWifiNlpBackend) - Uses Apple's service to resolve Wi-Fi locations. It has excellent coverage but the database is proprietary.
+* [Déjà Vu](https://github.com/n76/DejaVu) - A backend that uses locally acquired WLAN/WiFi AP and mobile/cellular tower data to resolve user location.
```

Comment:
FWIW: linking to the [wiki](https://github.com/microg/UnifiedNlp/wiki/Backends) would be easier.